src/common/utils/redis/helper_test.go

```diff
@@ -16,11 +16,12 @@ package redis
 
 import (
 	"fmt"
+	"github.com/goharbor/harbor/src/internal/cache"
```

Comment:
Maybe the go imports order can be adjusted?  built-in packages are in a section and 3rd packages could be in a section.

Just a suggestion.